AirCasting/SessionViews/ChartView.swift

```diff
@@ -23,44 +24,31 @@ class UI_PollutionChart: UIView {
             lineChartView.bottomAnchor.constraint(equalTo: self.bottomAnchor)
         ])
         
-        // Data
-        #warning("TODO: Replace mocked data with data from AirBeam")
-        let entries = [ChartDataEntry(x: 1, y: 3),
-                       ChartDataEntry(x: 2, y: 2),
-                       ChartDataEntry(x: 3, y: 1),
-                       ChartDataEntry(x: 4, y: 4)]
-        
-        let dataSet = LineChartDataSet(entries: entries)
-        let data = LineChartData(dataSet: dataSet)
-        lineChartView.data = data
-        
-        //format data labels
-        let formatter = NumberFormatter()
-        formatter.maximumFractionDigits = 0
-        data.setValueFormatter(DefaultValueFormatter(formatter: formatter))
-        data.setValueFont(UIFont(name: "Muli-Regular", size: 12)!)
-        data.setValueTextColor(UIColor(.aircastingGray))
-        
         //remove border lines and legend
-        lineChartView.xAxis.enabled = false
+        lineChartView.xAxis.enabled = true
         lineChartView.xAxis.drawLabelsEnabled = false
+        lineChartView.xAxis.labelPosition = .bottom
         lineChartView.xAxis.drawGridLinesEnabled = false
+        lineChartView.xAxis.axisMinimum = 0
+        lineChartView.xAxis.axisMaximum = 9
         
         lineChartView.leftAxis.drawLabelsEnabled = false
         lineChartView.leftAxis.drawAxisLineEnabled = false
         lineChartView.leftAxis.gridColor = UIColor(.aircastingGray).withAlphaComponent(0.4)
         
         lineChartView.rightAxis.enabled = false
-        lineChartView.rightAxis.drawLabelsEnabled = false
-        lineChartView.rightAxis.drawAxisLineEnabled = false
         
         lineChartView.legend.enabled = false
         
-        //dots colors
-        dataSet.circleHoleColor = UIColor(.aircastingGreen)
-        dataSet.setCircleColors(UIColor(.aircastingGreen).withAlphaComponent(0.5))
-        //line color
-        dataSet.setColor(UIColor(.aircastingGray).withAlphaComponent(0.7))
+        #warning("The text is not appearing and I don't know why")
+        lineChartView.noDataText = "Wait for the averages to appear"
+        lineChartView.noDataTextColor = .red
+        lineChartView.noDataTextAlignment = .center
```

Comment:
Ask Marysia to be sure, but you don't have to have this label at all. Looking at the designs, there's no expand button on the session cart when there are no measurements. You just display info that you're waiting for measurements. 
https://projects.invisionapp.com/d/main?origin=v7#/console/19745972/414476839/preview?scrollOffset=1920